develop/learning-paths/mobile/articles/06-using-your-screenlets/02-using-get-guestbooks-screenlet.markdown

```diff
@@ -0,0 +1,87 @@
+# Using Get Guestbooks Screenlet [](id=using-get-guestbooks-screenlet)
+
+The basic steps for using Get Guestbooks Screenlet are the same as those for 
+using any other Screenlet. Recall that you use a Screenlet by placing it in the 
+activity or fragment where you want it to appear. You'll use Get Guestbooks 
+Screenlet by inserting it in `GuestbooksActivity`. The following steps 
+illustrate this.
+
+## Inserting the Screenlet in the Layout [](id=inserting-the-screenlet-in-the-layout)
+
+When you used the Guestbook Mobile SDK directly, you displayed the guestbooks by 
+placing a `ListView` inside the `NavigationView` of `activity_guestbooks.xml`. 
+Since your Screenlet's View already contains a `ListView`, you no longer need 
+the one in `activity_guestbooks.xml`. Replace the `ListView` in 
+`activity_guestbooks.xml` with the following code:
+
+    <com.liferay.docs.getguestbooksscreenlet.GetGuestbooksScreenlet
+        android:id="@+id/getguestbooks_screenlet"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:paddingTop="@dimen/nav_header_height"
+        liferay:layoutId="@layout/guestbooks_default"/>
+
```

Comment:
In the `DrawerLayout` element (the document's first element), make sure to declare the `liferay` XML namespace by adding this attribute: `xmlns:liferay="http://schemas.android.com/apk/res-auto"`